mlflow/utils/helper_functions.py

```diff
@@ -0,0 +1,8 @@
+import platform
+
+
+def is_local_os_windows():
```

Comment:
```suggestion
def is_windows():
```

Can we rename this function to `is_windows`?